features-json/webp.json

```diff
@@ -48,7 +48,7 @@
       "15":"n",
       "16":"n",
       "17":"n",
-      "18":"n"
+      "18":"d"
```

Comment:
Change **d** to **n d** so as to be correct

Reply:
@FremyCompany Any way of fixing that ones I gave? It is needed to be done by now.